indico/modules/events/surveys/fields/base.py

```diff
@@ -50,6 +50,8 @@ class SurveyField(object):
     plugin = None
     #: displayed name of the field type
     friendly_name = None
+    #: wtform field class for this field
+    wtf_field_class = None
```

Comment:
True, I thought about that before, however I don't know why I hadn't done it that way..